Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN apt-get update && apt install -y libcap2-bin
 RUN setcap CAP_NET_BIND_SERVICE=+ep /usr/local/bin/envoy
 RUN setcap CAP_NET_BIND_SERVICE=+ep /usr/local/bin/$BIN_NAME
 
-FROM hashicorp/envoy-fips:1.29.4-fips1402 as envoy-fips-binary
+FROM hashicorp/envoy-fips:1.29.5-fips1402 as envoy-fips-binary
```

Comment:
This will require a new release of `hashicorp/envoy-fips` for the new version - I've added a note in the [runbook](https://go.hashi.co/runbook-envoy-dataplane) to call this out. @sarahalsmiller, @DanStough and I have both done this recently, so if you need pointers feel free to ask.